components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -405,9 +405,9 @@ export class ReportingSearchbarComponent implements OnInit {
     this.visibleDate.setFullYear(year);
   }
 
-  onDaySelect(date) {
+  onDaySelect(date: string) {
     this.visibleDate = new Date(date);
-    this.dateChanged.emit({ detail: new Date(this.visibleDate) });
+    this.dateChanged.emit({ detail: date });
```

Comment:
Sending the string '2019-10-23' instead of the date object which will be in local timezone.